docs/getting-started/download.md

```diff
@@ -28,10 +28,11 @@ Download the latest stable `hhfab` binary from the [GitHub Package](https://ghcr
 curl -fsSL https://i.hhdev.io/hhfab | bash
 ```
 
-Or download a specific version (e.g. beta-1) using the following command:
+Or download a specific version (e.g. 24.09) using the following command:
 
 ```bash
-curl -fsSL https://i.hhdev.io/hhfab | VERSION=beta-1 bash
```

Comment:
Let's keep the system wide install by default. If you think it makes sense you can add a separate note on how to install into current dir

Reply:
In walking through it with a customer, it was a smoother process bringing the binary into a local dir. But in the case of an upgrade it would be easier to overwrite an installed binary. Will do